src/Commands/SiteStudioColor.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Drupal\dst_entity_generate\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\dst_entity_generate\BaseEntityGenerate;
+use Drupal\dst_entity_generate\DstegConstants;
+
+/**
+ * Class provides functionality of Site Studio Integration for Color generation from DST sheet.
+ *
+ * @package Drupal\dst_entity_generate\Commands
+ */
+class SiteStudioColor extends BaseEntityGenerate {
+
+  /**
+   * {@inheritDoc}
+   */
+  protected $dstEntityName = 'site_studio_color';
+
+  /**
+   * Array of all dependent modules.
+   *
+   * @var array
+   */
+  protected $dependentModules = ['cohesion'];
+
+  /**
+   * Entity Type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * List of required fields to create entity.
+   *
+   * @var array
+   */
+  protected $requiredFields = ['id', 'machine_name'];
+
+  /**
+   * DstegMenu constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
+   *   Entity Type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager) {
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * Generate all the Site Studio Colors from DEG sheet.
+   *
+   * @command deg:generate:sscolor
+   * @aliases deg:ssc
+   * @usage drush deg:ssc
```

Comment:
Think about having one command for SiteStudio related or something like that... We shouldn't create a command for every setting in Site Studio IMO. Let's think of a better architecture and think it in the whole Site Studio rather then small bit of it.